locales/el/el.json

```diff
@@ -1,618 +1,618 @@
+    "Are you sure you would like to leave this team?": "Είσαι σίγουρος ότι θα ήθελες να φύγεις από αυτή την ομάδα;",
+    "Are you sure you would like to remove this person from the team?": "Είστε σίγουροι ότι θα θέλατε να αφαιρέσετε αυτό το άτομο από την ομάδα;",
+    "Argentina": "Αργεντινή",
+    "Armenia": "Αρμενία",
+    "Aruba": "Αρούμπα",
+    "Attach": "Επισυνάψετε",
+    "Attach & Attach Another": "Επισύναψη & Επισύναψη Άλλου",
+    "Attach :resource": "Επισύναψη :resource",
+    "August": "Αυγούστου",
+    "Australia": "Αυστραλία",
+    "Austria": "Αυστρία",
+    "Azerbaijan": "Αζερμπαϊτζάν",
+    "Bahamas": "Μπαχάμες",
+    "Bahrain": "Μπαχρέιν",
+    "Bangladesh": "Μπαγκλαντές",
+    "Barbados": "Μπαρμπάντος",
     "Before proceeding, please check your email for a verification link.": "Πριν συνεχίσετε, ελέγξτε το email σας για έναν σύνδεσμο επαλήθευσης.",
-    "Belarus": "Belarus",
-    "Belgium": "Belgium",
-    "Belize": "Belize",
-    "Benin": "Benin",
-    "Bermuda": "Bermuda",
-    "Bhutan": "Bhutan",
-    "Bolivia": "Bolivia",
-    "Bonaire, Sint Eustatius and Saba": "Bonaire, Sint Eustatius and Saba",
-    "Bosnia And Herzegovina": "Bosnia and Herzegovina",
-    "Botswana": "Botswana",
-    "Bouvet Island": "Bouvet Island",
-    "Brazil": "Brazil",
-    "British Indian Ocean Territory": "British Indian Ocean Territory",
-    "Browser Sessions": "Browser Sessions",
+    "Belarus": "Λευκορωσία",
+    "Belgium": "Βέλγιο",
+    "Belize": "Μπελίζ",
+    "Benin": "Μπενίν",
+    "Bermuda": "Βερμούδα",
+    "Bhutan": "Μπουτάν",
+    "Bolivia": "Βολιβία",
+    "Bonaire, Sint Eustatius and Saba": "Bonaire, Sint Eustatius και Sábado",
+    "Bosnia And Herzegovina": "Βοσνία-Ερζεγοβίνη",
+    "Botswana": "Μποτσουάνα",
+    "Bouvet Island": "Νήσος Μπουβέ",
+    "Brazil": "Βραζιλία",
+    "British Indian Ocean Territory": "Βρετανικό Έδαφος Ινδικού Ωκεανού",
+    "Browser Sessions": "Συνεδρίες Προγράμματος Περιήγησης",
     "Brunei Darussalam": "Brunei",
-    "Bulgaria": "Bulgaria",
-    "Burkina Faso": "Burkina Faso",
-    "Burundi": "Burundi",
-    "Cambodia": "Cambodia",
-    "Cameroon": "Cameroon",
-    "Canada": "Canada",
-    "Cancel": "Cancel",
-    "Cape Verde": "Cape Verde",
-    "Card": "Card",
-    "Cayman Islands": "Cayman Islands",
-    "Central African Republic": "Central African Republic",
-    "Chad": "Chad",
-    "Changes": "Changes",
-    "Chile": "Chile",
-    "China": "China",
-    "Choose": "Choose",
-    "Choose :field": "Choose :field",
-    "Choose :resource": "Choose :resource",
-    "Choose an option": "Choose an option",
-    "Choose date": "Choose date",
-    "Choose File": "Choose File",
-    "Choose Type": "Choose Type",
-    "Christmas Island": "Christmas Island",
+    "Bulgaria": "Βουλγαρία",
+    "Burkina Faso": "Μπουρκίνα Φάσο",
+    "Burundi": "Μπουρούντι",
+    "Cambodia": "Καμπότζη",
+    "Cameroon": "Καμερούν",
+    "Canada": "Καναδά",
+    "Cancel": "Ακύρωση",
+    "Cape Verde": "ακρωτήριο",
```

Comment:
Totally wrong
It is officially known as "[Δημοκρατία του Κάμπου Βέρντε](https://el.wikipedia.org/wiki/%CE%A0%CF%81%CE%AC%CF%83%CE%B9%CE%BD%CE%BF_%CE%91%CE%BA%CF%81%CF%89%CF%84%CE%AE%CF%81%CE%B9%CE%BF)"